packages/core/src/components/Table/TableCell/TableCell.styles.tsx

```diff
@@ -0,0 +1,162 @@
+import { CSSInterpolation } from "@emotion/serialize";
+import { alpha, hexToRgb } from "@mui/material";
+import { theme } from "@hitachivantara/uikit-styles";
+import { CSSProperties } from "react";
+import tableCellClasses from "./tableCellClasses";
+
+export const styles: { [key: string]: CSSInterpolation } = {
+  root: {
+    verticalAlign: "inherit",
+    textAlign: "left",
+    padding: `${theme.table.cellPaddingTop} ${theme.space.xs} ${
+      theme.table.cellPaddingBottom
+    } ${theme.spacing(4)}`,
+    borderBottom: `1px solid ${theme.colors.atmo4}`,
+  },
+
+  head: {
+    height: 52,
+    verticalAlign: "top",
+
+    backgroundColor: theme.colors.atmo1,
+    borderTop: `1px solid ${theme.table.headerBorderTopColor}`,
+    borderBottom: `1px solid ${theme.colors.atmo4}`,
+    ...(theme.typography.label as CSSProperties),
+  },
+  body: {
+    minHeight: 32,
+    "td&": {
+      height: 32,
+    },
+    backgroundColor: "inherit",
+    ...(theme.typography.body as CSSProperties),
+    fontFamily: theme.fontFamily.body,
+
+    [`&.${tableCellClasses.sorted}`]: {
+      backgroundColor: alpha(hexToRgb(theme.colors.atmo1), 0.4),
```

Comment:
Since `theme.colors.atmo1` is a CSS variable, it won't work with the `hexToRgb` method. This is the background color created:

```
rgba(10, NaN, NaN, 0.4, NaN, -0.047, NaN, NaN, NaN, 0.039, NaN, 0.004);
```

We need to use real values that are available in the React context side. 

This is happening with in order place in the table cell and table row.